src/sentry/static/sentry/app/routes.jsx

```diff
@@ -339,6 +329,24 @@ function routes() {
     hooksOrgRoutes.push(cb());
   });
 
+  // Add members route in the function because it needs the hook store
+  // which is not available at import time.
+  orgSettingsRoutes.push(
```

Comment:
Hmm, now that I think about it, may be better to move the array inside of the `routes` function so that it's deterministic, in the very unlikely chance that `routes` gets called multiple times. Sorry about that

Reply:
Oh you're totally right